provider/oauth2/tests.py

```diff
@@ -597,6 +597,25 @@ def test_access_token_response_valid_token_type(self):
         token = self._login_authorize_get_token()
         self.assertEqual(token['token_type'], constants.TOKEN_TYPE, token)
 
+    def test_client_credentials_grant(self):
+        response = self.client.post(self.access_token_url(), {
+            'grant_type': 'client_credentials',
+            'client_id': self.get_client().client_id,
+            'client_secret': self.get_client().client_secret,
+        })
+
+        self.assertEqual(200, response.status_code, response.content)
```

Comment:
I'm not sure I follow.  Perhaps I missed an earlier conversation on this.  
But the RFC specifically states in Section 4.4.3 for the Client Credentials Grant:
```
A refresh token SHOULD NOT be included.
```
So, if the Client Credentials grant is implemented correctly, then we should make sure in the test that we don't return a `refresh_token`.